tests/lib/tools/tests.invariant

```diff
@@ -142,6 +142,22 @@ check_broken_snaps() {
 	fi
 }
 
+check_cgroup_scopes() {
+	# xenial has leftover scope files possibly due to an old systemd bug
+	# https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1934147
+	if [ "$SPREAD_SYSTEM" == "ubuntu-core-16-64" ] || [ "$SPREAD_SYSTEM" == "ubuntu-16.04-64" ]; then
```

Comment:
Is this equivalent of doing `os-query is-xenial`?

Reply:
Not for core at least

Reply:
This probably needs a `ubuntu-core-16-64` -> `ubuntu-core-16-*` so that it still works when we run this in the cert lab on other arches than amd64 (the suffix there is e.g. `-arm-64`)

Reply:
And of course this needs `[[` around it when using with globs (thank you shell!). I will tweak and push.

Reply:
Ah I see, thanks

Reply:
And of course no `""` inside `[[ ]]` for globbing to work - again - THANK YOU shell for being full of fun special cases.